packages/stacked_tools/lib/src/models/config_model.dart

```diff
@@ -46,6 +46,14 @@ class Config with _$Config {
     @JsonKey(name: 'register_mocks_function')
     @Default('registerServices')
         String registerMocksFunction,
+
+    /// Boolean value to determine view builder style
+    ///
+    /// This is used when creating a view file during `create view` command. By
+    /// default, ViewModelWidget style is used.
+    @JsonKey(name: 'use_view_model_builder_style')
```

Comment:
I think we should shorten this. I prefer something like

```shell
stacked create view login --v1
```

or to make it clearer

```shell
stacked create view login --useBuilder
```

@ferrarafer this actually reminds me that we can make the name better for the new style widget. I think we should call it `StackedView<T>` or ``ViewModelView<T>` I like both. One builds our brand, the other is aligned with MVVM. 

Let me know what you think

Reply:
I like StackedView but we probably need to update Service to StackedService, isn't it?